register/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from .views import CreateUserView
+
+app_name = "register"
+
+urlpatterns = [
+    path("", CreateUserView.as_view(), name="create_user")
```

Comment:
empty path? Does it mean that it will be our main page? Or it will be in /register?

Reply:
it will be in /api/register/